src/PhysicalOutbox.StorageTable.AcceptanceTests/ConfigureAzureTablePersistence.cs

```diff
@@ -32,8 +32,10 @@ Task IConfigureEndpointTestExecution.Configure(string endpointName, EndpointConf
             }
 
             // This populates the partition key at the physical stage to test the conventional outbox use-case
-            configuration.Pipeline.Register(typeof(PartitionKeyProviderBehavior), "Populates the partition key");
-
+            if (endpointName != Conventions.EndpointNamingConvention(typeof(When_using_outbox_control_message.Endpoint)))
```

Comment:
Not proud of this but similar to line 29 above

Reply:
In Cosmos this worked really well

https://github.com/Particular/NServiceBus.Persistence.CosmosDB/blob/master/src/SharedAcceptanceTests.RequirePartitionKey/RunSettingsExtensionsRequirePartitionKey.cs#L7-L8

https://github.com/Particular/NServiceBus.Persistence.CosmosDB/blob/8c8a04078d00415ad83ddc4b7d1465fff3690c47/src/NServiceBus.Persistence.CosmosDB.PhysicalOutbox.AcceptanceTests/ConfigureEndpointCosmosDBPersistence.cs#L26-L33

https://github.com/Particular/NServiceBus.Persistence.CosmosDB/blob/8c8a04078d00415ad83ddc4b7d1465fff3690c47/src/NServiceBus.Persistence.CosmosDB.LogicalOutbox.AcceptanceTests/When_custom_partition_key_extractor_from_message_registered_via_di.cs#L19-L20

Reply:
Thanks, fixed now